study-builder/studies/pancan/study-emails.conf

```diff
@@ -15,6 +15,14 @@
       "filepath": "emails/about_you_completed.html",
       "render": true,
       "isDynamic": true
+    },
+    {
+      "key": "consentCompleted",
+      "name": "PANCAN_CONSENT_COMPLETE",
+      "subject": "Thank you for providing your consent",
+      "filepath": "emails/consent-completed.html",
```

Comment:
I guess it is worth using an English language template here 
`emails/consent-completed.en.html`

Reply:
I thought there were smarts in the server which resolved the filename down to the proper template for each language, but it looks like `TestBoston` is using the pattern you've mentioned here. I'll get the change in!

Reply:
Fixed in 5be03ae6c002cfc27b09917be174a8ac5f5fb028